### PR TITLE
Adding caption control

### PR DESCRIPTION
**Pull request by alanlanglois**
Adding caption control next to the timer to be able to activate or disable caption on react-native-video. 

image

### CODE REVIEW

File: VideoPlayer.js [hunk 1]
```diff
@@ -1061,6 +1078,18 @@ export default class VideoPlayer extends Component {
         );
     }
 
+    /**
+     * Render the seekbar and attach its handlers
```
Comment (VincentCATILLON) [inline VideoPlayer.js]: Be careful about the description, it's not the seekbar

Comment (alanlanglois) [reply VideoPlayer.js]: you're right thx for the feedback 👍

**Comment by VincentCATILLON** [comment]
Is this one duplicate of #132 ?

**Comment by kylemilloy** [comment]
See notes on https://github.com/itsnubix/react-native-video-controls/pull/132

**Pull request closed without merging**